frontend/src/components/Form/Toast.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+import styled, { css } from "styled-components";
+
+interface iToast {
+  type: string;
+  isActive?: boolean;
+  children?: React.ReactNode;
+  isVisible?: boolean;
+}
+const Toast: FC<iToast> = ({ type, isActive, children }) => {
+  return (
+    <Container type={type} isVisible={isActive}>
+      {children}
+    </Container>
+  );
+};
+
+const Container = styled.section`
+  position: fixed;
+  width: 100%;
+  padding: 20px;
+  text-align: center;
+  color: white;
+  background-color: orange;
+  ${(p: iToast) =>
+    p.isVisible &&
+    css`
+      top: 0;
+    `};
+  ${(p: iToast) =>
+    !p.isVisible &&
+    css`
+      top: -100%;
+    `};
```

Comment:
`p`? What does `p` mean? Use descriptive names so every one can understand what it is.

This code would be like this:
```
top: ${({ isVisible }) => isVisible ? 0 : '-100%'}
```
(note that `'-100%'` is between quotes (a string)